### PR TITLE
zh:add tidbmonitor dynamic configuration

### PR DESCRIPTION
**Pull request by mikechengwei**
### What is changed, added, or deleted? (Required)

Add tidbmonitor dynamic configuration doc.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by csuzhangxc — APPROVED** [review]

File: zh/enable-monitor-dynamic-configuration.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+summary: 动态更新 TidbMonitor 配置
+---
+
+# 开启动态配置功能
+
+本文档介绍如何开启 TidbMonitor 动态配置功能。
+
+## 功能介绍
+
+TidbMonitor 支持多集群，分片等功能，Prometheus的配置，Rule，Targets 变更，如果不开启动态配置需要重启才能生效，如果监控数据量很大，恢复 Prometheus 快照数据耗时会比较长。
```
Comment (qiancai) [inline zh/enable-monitor-dynamic-configuration.md:12]:
```suggestion
TidbMonitor 支持多集群、分片等功能，当 Prometheus的配置、Rule、Targets 变更时，如果不开启动态配置，这些变更只能在重启后才能生效。如果监控数据量很大，重启后恢复 Prometheus 快照数据耗时会比较长。
```

Comment (qiancai) [reply zh/enable-monitor-dynamic-configuration.md:12]: 请问这里的重启是指 TidbMonitor 重启吗？

File: zh/enable-monitor-dynamic-configuration.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+当开启动态配置功能时，TidbMonitor 的配置都可以做到热更新。
```
Comment (qiancai) [inline zh/enable-monitor-dynamic-configuration.md:14]:
```suggestion
开启动态配置功能后，TidbMonitor 的配置更改即可动态更新。
```

File: zh/enable-monitor-dynamic-configuration.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## 如何开启动态配置功能
+
+开启动态配置功能，需要指定 `prometheusReloader` 配置，示例如下:
```
Comment (qiancai) [inline zh/enable-monitor-dynamic-configuration.md:18]:
```suggestion
在 TidbMonitor 的 spec 中，你可以通过指定 `prometheusReloader` 开启动态配置功能。示例如下:
```

File: zh/enable-monitor-dynamic-configuration.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      namespace: ns1
+    - name: ns2
+      namespace: ns2
+  prometheusReloader:
+    baseImage: quay.io/prometheus-operator/prometheus-config-reloader
+    version: v0.49.0
+  imagePullPolicy: IfNotPresent
+```
+
+变更后 TidbMonitor 会发生重启，后续针对 Prometheus 的配置变更都会动态更新。
```
Comment (qiancai) [inline zh/enable-monitor-dynamic-configuration.md:38]:
```suggestion
`prometheusReloader` 配置变更后，TidbMonitor 会自动重启。重启后，所有针对 Prometheus 的配置变更都会动态更新。
```

File: zh/enable-monitor-dynamic-configuration.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+可以参考 [Example](https://github.com/pingcap/tidb-operator/tree/master/examples/monitor-dynamic-configmap)。
```
Comment (qiancai) [inline zh/enable-monitor-dynamic-configuration.md:40]:
```suggestion
可以参考 [monitor-dynamic-configmap 配置示例](https://github.com/pingcap/tidb-operator/tree/master/examples/monitor-dynamic-configmap)。
```

File: zh/enable-monitor-dynamic-configuration.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+本文档介绍如何开启 TidbMonitor 动态配置功能。
+
+## 功能介绍
+
+TidbMonitor 支持多集群，分片等功能，Prometheus的配置，Rule，Targets 变更，如果不开启动态配置需要重启才能生效，如果监控数据量很大，恢复 Prometheus 快照数据耗时会比较长。
+
+当开启动态配置功能时，TidbMonitor 的配置都可以做到热更新。
+
+## 如何开启动态配置功能
```
Comment (qiancai) [inline zh/enable-monitor-dynamic-configuration.md:16]: 开启后如何关闭可以也说下吗？是删除 `prometheusReloader` 配置就可以吗？

File: zh/enable-monitor-dynamic-configuration.md (hunk 1 above)
Comment (mikechengwei) [reply zh/enable-monitor-dynamic-configuration.md:12]: 是的

File: zh/enable-monitor-dynamic-configuration.md (hunk 6 above)
Comment (mikechengwei) [reply zh/enable-monitor-dynamic-configuration.md:16]: 是的，我补充下

File: zh/enable-monitor-dynamic-configuration.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+summary: 动态更新 TidbMonitor 配置
+---
+
+# 开启动态配置功能
+
+本文档介绍如何开启 TidbMonitor 动态配置功能。
+
+## 功能介绍
+
+TidbMonitor 支持多集群、分片等功能，当 Prometheus的配置、Rule、Targets 变更时，如果不开启动态配置，这些变更只能在重启后才能生效。如果监控数据量很大，重启后恢复 Prometheus 快照数据耗时会比较长。
```
Comment (qiancai) [inline zh/enable-monitor-dynamic-configuration.md:12]:
```suggestion
TidbMonitor 支持多集群、分片等功能，当 Prometheus 的配置、Rule、Targets 变更时，如果不开启动态配置，这些变更只能在重启后才能生效。如果监控数据量很大，重启后恢复 Prometheus 快照数据耗时会比较长。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**